Product/Production/Adapters/General/CONNECTAdminGUI/src/main/java/gov/hhs/fha/nhinc/admingui/managed/LoginBean.java

```diff
@@ -72,7 +72,7 @@
      */
     public Boolean isCorrect = false;
 
-    private static boolean firstTimeLogged = true;
+    private boolean firstTimeLogged = true;
```

Comment:
Another option is to make checkDisplays() static.  If we go with that we would have to to change login and loginAndNavigate to static, to pass the next Sonar scan. I don't know if that would bring up a problem since loginAndNavigate() is called from login.xhtml.